DepositeCalcTests/Pages/SettingsPage.cs

```diff
@@ -0,0 +1,57 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DepositeCalcTests.Pages
+{
+    internal class SettingsPage
+    {
+        private readonly IWebDriver driver;
+        public SettingsPage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+        private IWebElement DateFormatFld => driver.FindElement(By.XPath("//select[@id='dateFormat']"));
+        private IWebElement NumberFormatFld => driver.FindElement(By.XPath("//select[@id='numberFormat']"));
+        private IWebElement DefaultCurrencyFld => driver.FindElement(By.XPath("//select[@id='currency']"));
+        private IWebElement SaveBnt => driver.FindElement(By.XPath("//button[@id='save']"));
+        private IWebElement CancelBtn => driver.FindElement(By.XPath("//button[@id='cancel']"));
+        private IWebElement LogoutLnk => driver.FindElement(By.XPath("//div[@onclick='Logout()']"));
+
+        public void ClickOnLogoutLnk()
+        {
+            LogoutLnk.Click();
+        }
+
+        public void ClickOnCancelBtn()
+        {
+            CancelBtn.Click();
+        }
+
+        public void ClickOnSaveBnt()
```

Comment:
Let it be just `Save`. 
```suggestion
        public void Save()
```